app/workers/asset_manager_create_whitehall_asset_worker.rb

```diff
@@ -3,18 +3,18 @@ class AssetManagerCreateWhitehallAssetWorker < WorkerBase
 
   sidekiq_options queue: "asset_manager"
 
-  def perform(file_path, legacy_url_path, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [])
+  def perform(file_path, legacy_url_path, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [], access_limited_organisation_ids = [])
     return unless File.exist?(file_path)
 
     file = File.open(file_path)
-    asset_options = { file:, legacy_url_path:, auth_bypass_ids: }
+    asset_options = { file:, legacy_url_path:, auth_bypass_ids:, access_limited_organisation_ids: }
```

Comment:
Is this needed if the worker previously didn't accept `access_limited` user IDs?